graphblas/io.py

```diff
@@ -455,7 +455,7 @@ def to_awkward(A, format=None):
         indices, values = A.to_coo()
         form = RecordForm(
             contents=[
-                NumpyForm(A.dtype.numba_type.name, form_key="node1"),
+                NumpyForm(A.dtype.numba_type.name, form_key="node1"),  # XXX: is numba required?
```

Comment:
@jim22k, is there a reason `A.dtype.numba_type.name` is used instead of `A.dtype.np_type.name`? Numba doesn't appear to be a strict requirement for Awkward.

Reply:
Using `A.dtype.np_type.name` seems to work; I'm going to assume this resolves it and works fine.